test/BUILD.gn

```diff
@@ -60,6 +60,12 @@ test("brave_unit_tests") {
     "../utility/importer/firefox_importer_unittest.cc",
   ]
 
+  if (brave_rewards_enabled) {
+    sources += [
```

Comment:
I don't know if we want to include tests from the library here. I think it should have its own independent test runner, but maybe this is ok for now as long as their is a ticket open to add testing support to the library directly
cc @bbondy

Reply:
https://github.com/brave-intl/bat-native-ledger/issues/126